backends/blocked/ceed-blocked-operator.c

```diff
@@ -40,7 +40,9 @@ static int CeedOperatorDestroy_Blocked(CeedOperator op) {
 
   for (CeedInt i=0; i<impl->numeout; i++) {
     ierr = CeedVectorDestroy(&impl->evecsout[i]); CeedChk(ierr);
-    ierr = CeedVectorDestroy(&impl->qvecsout[i]); CeedChk(ierr);
+    if (!impl->identityqf) {
+      ierr = CeedVectorDestroy(&impl->qvecsout[i]); CeedChk(ierr);
+    }
```

Comment:
This is okay, but perhaps we should have `qvecsout_alloc[]` so that destruction responsibilities are clear up-front?